index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title></title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="index.css">
+        <script src="https://kit.fontawesome.com/5031df2cf9.js" crossorigin="anonymous"></script>
+    </head>
+    <body>
+        <header>
+          <nav>
+            <div class="logo">
+                My Logo
+            </div>
```

Comment:
- [ ] As the previous reviewer has earlier suggested, I think it will be better if you make your logo clickable by wrapping it in an anchor <a> tag so that it will be easy for users to navigate through your website which is a very good user experience 😄